app/routes/query.$version.docs.framework.$framework.$.tsx

```diff
@@ -10,12 +10,21 @@ export const loader = async (context: LoaderFunctionArgs) => {
   const { '*': docsPath, framework, version } = context.params
   const { url } = context.request
 
+  let redirectPath = url.replace(/\/docs.*/, '/docs/framework/react/overview')
+
+  if (url.includes(`docs/framework/${framework}/reference`)) {
+    redirectPath = url.replace(
+      `docs/framework/${framework}/reference`,
+      'docs/reference'
+    )
+  }
+
   return loadDocs({
     repo,
     branch: getBranch(version),
     docPath: `docs/framework/${framework}/${docsPath}`,
     currentPath: url,
-    redirectPath: url.replace(/\/docs.*/, '/docs/framework/react/overview'),
```

Comment:
I think we could just replace this with  `url.replace(`/framework/${framework}/`, '')` - if any framework-specific route fails to match, it would try redirecting to the core equivalent, which would then redirect to the overview doc from that loader if that also failed.

Reply:
Sorry that code should actually be:
```
url.replace(`/framework/${framework}/`, '/')
```

Reply:
I like it. Tested it locally, and it seems to work!